docs/src/main/paradox/docs/delta/api/views/composite-sink.md

```diff
@@ -0,0 +1,173 @@
+# Composite Sinks
+
+A Composite Sink handles the following steps of composite view indexing
+
+* Querying for the graphs of resources in the Blazegraph common namespace of the view
+* Converting the obtained graphs into a format that can be pushed to a target sink
+* Finally, pushing the resources to the target sink
+
+These steps can be implemented in different ways. In Nexus Delta, there are two kinds of Composite Sink that can be
+selected via configuration.
+
+1. @ref:[Single Composite Sink](#single-composite-sink)
+2. @ref:[Batch Composite Sink](#batch-composite-sink)
+
+## Single Composite Sink
+
+By default, Nexus Delta will use the Single Composite Sink. This sink performs one query to the Blazegraph common namespace for each resource in the project. The queries are done in chronological order (by the `updatedAt` time of the resources).
+
+We recommend reading through the @ref:[search configuration example use case](../../../getting-started/running-nexus/search-configuration.md#example-use-case) and the @ref[Composite View API reference](composite-view-api.md) to learn more about Composite Views.
+
+## Batch Composite Sink
+
+@@@ note { .warning }
+
+We recommend to start using Composite Views with the default @ref:[Single Composite Sink](#single-composite-sink). Once
+you have a good understanding of it, the Batch Composite Sink can be used to enhance the performance of your deployment.
+
+@@@
```

Comment:
I think the warning needs to come after the description

Reply:
Changed